sycl/include/sycl/ext/oneapi/experimental/device_architecture.hpp

```diff
@@ -124,12 +152,37 @@ enum class architecture {
   intel_gpu_12_10_0 = intel_gpu_dg1,
 };
 
+enum class arch_category {
```

Comment:
I expect that we will add  new categories soon.  Should there be a comment here explaining what to do when a new category is added, similar to the comment you have above explaining what to do when a new enumerator us added to `enum class architecture`?

Reply:
Good point, added in https://github.com/intel/llvm/pull/12752/commits/5a8efb0ca2b8a44dd4bace4d3163415949bda28f